Makefile

```
@@ -160,3 +160,8 @@ release.prod: test
 .PHONY: ko
 ko:
 	scripts/install-ko.sh
+
+# generate-documentation: Generate documentation (docs/flags.md)
+.PHONE: generate-documentation
+generate-documentation:
```

What do you think of: 


```suggestion
.PHONE: generate-flags-documentation
generate-flags-documentation:
```